zombienet_tests/async_backing/async-backing-test.toml

```diff
@@ -0,0 +1,32 @@
+[settings]
+timeout = 1000
+
+[relaychain]
+default_image = "{{ZOMBIENET_INTEGRATION_TEST_IMAGE}}"
+chain = "rococo-local"
+default_command = "polkadot"
+
+  [relaychain.genesis.runtime.runtime_genesis_config.configuration.config]
+  max_validators_per_core = 2
```

Comment:
This is redundant in this test. We have 1 parachain and 2 validators. This runtime configuration parameter sets the maximum backing group size, which will default to no maximum in this case, hence size is 2.